packages/cadl-ranch-specs/http/models/optional-properties/mockapi.ts

```diff
@@ -0,0 +1,53 @@
+import { passOnSuccess, mockapi, ValidationError, json } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.OptionalProperties_sendOptionalPropertyModel = passOnSuccess(
```

Comment:
For example here I would expect the scenarios for a model with optional properties to be:
- Send only required properties(Mock api can just call `req.expectBodyEquals()`) which should make sure the optional properrties are not send with `null`
- Receive only required properties
- Send with optional properties
- Receive with optional properties

Reply:
> Send only required properties(Mock api can just call req.expectBodyEquals()) which should make sure the optional properrties are not send with null

There is no required properties in the models. And I already checked the properties in below `checkPropertyValid`. What do you mean by `required properties`.

Reply:
I see, so the goal of this is to test a model with different optional property types. Maybe then should be named `OptionalPropertyTypes` and then test should describe which type they are checking work fine as am optional property